src/testdrive/src/action/s3.rs

```diff
@@ -16,27 +16,43 @@ pub async fn run_verify_data(
     mut cmd: BuiltinCommand,
     state: &State,
 ) -> Result<ControlFlow, anyhow::Error> {
-    let expected_body = cmd.input.join("\n") + "\n";
+    let expected_body = cmd.input.sort();
```

Comment:
There is precedent in testdrive that if sorting is needed to validate results it is an explicit flag. Check the `sort-messages=true` flag for `kafka-verify`. Should we do the same here?

Having the ability to verify data without sorting will allow us to write a testdrive that does `COPY TO S3` with a single replica cluster and then verify that we write the data in the correct order in the file

Reply:
@nrainer-materialize @def- what do you think?

Reply:
I agree with Petros.